### DISCUSSION TITLE
The French "new communes" are missing

### DISCUSSION THREAD

**Discussion by Rpnpif** [breezy-weather/breezy-weather#978] [Ideas]
### Describe your idea

New French communes (_nouvelles communes_) are being created in France from the grouping of ancient communes since 2015 or before.
Their names are being more and more used for location. The ancient communes names are still used but the maps forgot them more and more.
The new communes names are missing in Breezy Weather. So please add them perhaps in addition of ancient communes names.

### Other details

Nominatim or other from OpenStreetMap have an API to retrieve these names

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [x] I have updated the app to **[version 5.2.0 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by Rpnpif** [comment]
[QuickWeather](https://github.com/TylerWilliamson/QuickWeather) knows how to use new French communes.

**Comment by papjul** [comment]
What's your location search source? Contact them to let them know their database is outdated

We don't use Nominatim because they don't want to add timezone to their database, and it's a mandatory info for us
